src/resolution_functions/instrument.py

```diff
@@ -355,7 +357,7 @@ def _get_file(instrument_name: str) -> tuple[str, Union[str, None]]:
         except KeyError:
             raise InvalidInstrumentError(
                 f'"{instrument_name}" is not a valid instrument name. Only the following instruments are '
-                f'supported: {list(INSTRUMENT_MAP.keys())}')
+                f'supported: {list(INSTRUMENT_MAP)}')
```

Comment:
Could do
```suggestion
                f'supported: {', '.join(INSTRUMENT_MAP)}')
```
For slightly neater output. Fine as-is.

Reply:
I think I like having the brackets there a bit better, but maybe I'm weird